test/DepositManager.test.js

```diff
@@ -0,0 +1,129 @@
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import chaiBigNumber from 'chai-bignumber'
+import BigNumber from 'bignumber.js'
+import BN from 'bn.js'
+
+import deployer from './helpers/deployer.js'
+import * as _contracts from './helpers/contracts.js'
+import logDecoder from './helpers/log-decoder.js'
+import { assertBigNumberEquality, assertBigNumbergt } from './helpers/utils.js'
+
+chai
+  .use(chaiAsPromised)
+  // .use(chaiBigNumber(web3.BigNumber))
```

Comment:
@atvanguard  why are we using two different big number libs? can't we settle on single one.

Reply:
Cleaned up the mess :D

Reply:
for in-test usage, we can just use `web2.utils.BN` 
please remove imports of `BN.js and BigNumber.js`.
since all helpers have web3 instance injected i would prefer using `web3.utils.BN` there as well.
what do you think? @atvanguard

Reply:
Yo bro, I clean all this up. Check now.

Reply:
thanks